goodmap/data_validator.py

```diff
@@ -0,0 +1,85 @@
+import json
+from sys import argv, stderr
+
+
+def get_pts_with_missing_obligatory_fields(datapoints, obligatory_fields):
```

Comment:
Just came with tought. You're not collection missing points which miss obligatory fields. You're collecting `data_violations`.

Consider making such output - list of objects named `DataViolation` with enum ViolationType, Point, Field/ViolatingAtribute.